openreview/venue_request/venue_request.py

```diff
@@ -1449,14 +1455,16 @@ def setup_request_form(self):
                             'values-copied': [
                                 self.support_group.id,
                                 '{signatures}',
-                                '{content["program_chair_emails"]}'
+                                '{content["program_chair_emails"]}',
+                                '{content["publication_chair_email"]}'
```

Comment:
can you debug the UI and see what happens when we post the note?

is it being sent a "null" value? if that is the case, we should check in view.js where we are getting the copied values and if the value is null then don't send it.